db-experiment/Vagrantfile

```diff
@@ -0,0 +1,28 @@
+# -*- mode: ruby -*-
+
+box = "generic/ubuntu1804"
+ram = "65536"
```

Comment:
unused constant

Reply:
Use for the `hyperv.maxmemory` but value same as `hyperv.memory`. Copied this from another VM setup, I'm not sure how they are different. Wes do you know?

Reply:
hyperv.memory is allocated at startup, whereas hyperv.maxmemory is the max limit. If both are set, I'd set them to be the same - I'm not convinced dynamic memory allocation will help us on this host...

...but in that case, possibly, we should just not set maxmemory at all, which makes hyperv.memory work as a static allocation anyway.

Reply:
So I'm not 100% sure why both are set - there might have been a bug in the config handling of earlier vagrant, which forced me to set them both even if I only wanted to set static.